sparkbox/safe-focus
# Chore:  Setup GitHub actions

⚠️ Do not merge this PR before dropping commit d2bfc06fceb91d6732ddfe8a28412f365b87f79e! This commit is only used during the validation process.

This PR updates the publish script used with GitHub Actions.

### Validation Steps
1. [Verify that "NPM Publish"](https://github.com/sparkbox/safe-focus/runs/452855427?check_suite_focus=true) appears when the version of the package is greater than 1.1.0.
2. [Verify that "Skipping publish"](https://github.com/sparkbox/safe-focus/runs/452849183?check_suite_focus=true) appears when the version of the package is equal to 1.1.0.
